redis/keyspace_utils.cc

```diff
@@ -179,8 +179,9 @@ future<> create_keyspace_if_not_exists_impl(db::config& config, int default_repl
         logger.info("Create keyspace: {}, table: {} for redis.", ks_name, cf_name);
         return service::get_local_migration_manager().announce_new_column_family(schema, false);
     };
-    // create 16 default database for redis.
-    return parallel_for_each(boost::irange<unsigned>(0, 16), [keyspace_gen = std::move(keyspace_gen), table_gen = std::move(table_gen)] (auto c) {
+    // Create keyspaces as configured by redis_database_count, which are used as redis databases.
+    return parallel_for_each(boost::irange<unsigned>(0, config.redis_database_count()),
+                             [keyspace_gen = std::move(keyspace_gen), table_gen = std::move(table_gen)] (auto c) {
         auto ks_name = sprint("REDIS_%d", c);
         return keyspace_gen(ks_name).then([ks_name, table_gen] {
             return when_all_succeed(
```

Comment:
The seastar change (below) is broken. Never ever use `git add -a` and be careful about what you commit.

Dequeued this series.

Reply:
Thanks for the reminder.
This PR isn't fix something serious, I will put myself in confinement some days, then resend a correct one.